### DISCUSSION TITLE
This is an Absolutely bonkers mod

### DISCUSSION THREAD

**Discussion by DurpDaCrooze** [BudgetAirpods/InsanityRemastered#12] [General]
Please keep updating this mod, it's insane and like I'd genuinely donate if need be to see this mod expand. Kudos to the dev, had the living shit scared out of me after getting locked in a 1x1 room for an entire game as I descended into madness and started seeing my "comrades" walk around me.

**Comment by BudgetAirpods** [comment]
I appreciate your comment! Unless you can donate more free-time for me to work on the mod, I won't be accepting donations at this moment.

I always perceive things I make as garbage and not good enough. This led me to question whether I should scrap the mod entirely or let someone else take over. Seeing this comment just makes me want to work even harder on the mod. I thank you for that.
